README.md

```diff
@@ -1,3 +1,54 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Installation
+[Get Composer](https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md)
+
+```bash
+composer require --dev  mollie/php-coding-standards:v1.0
+```
+
+## Usage
+
+### Manual Triggering
+Run following command in your project directory, that will run fixer for every `.php` file.
+```bash
+vendor/bin/php-cs-fixer fix .
+```
+
+### Use via PHPStorm file watcher
+Please follow [official PHPStorm documentation](https://www.jetbrains.com/help/phpstorm/using-php-cs-fixer.html#f21a70ca)
+
+### Use via pre-commit, pre-push etc. git hook
```

Comment:
Please remove all notions of `pre-push` here, as the bash code purposely is a `pre-commit` hook. Pre-push it will do nothing because there will be no staged changes and doing a `git add` wouldn't do anything either.